### PR TITLE
workflows/tests: use envs rather than interpolation

### PR DESCRIPTION
**Pull request by Bo98**
This is generally safer practice.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -151,12 +155,12 @@ jobs:
             core.setOutput('container', JSON.stringify(container))
 
             const test_bot_formulae_args = ["--only-formulae", "--junit", "--only-json-tab", "--skip-dependents"]
-            test_bot_formulae_args.push('--testing-formulae=${{needs.tap_syntax.outputs.testing_formulae}}')
-            test_bot_formulae_args.push('--added-formulae=${{needs.tap_syntax.outputs.added_formulae}}')
-            test_bot_formulae_args.push('--deleted-formulae=${{needs.tap_syntax.outputs.deleted_formulae}}')
+            test_bot_formulae_args.push(`--testing-formulae=${process.env.TESTING_FORMULAE}`)
+            test_bot_formulae_args.push(`--added-formulae=${process.env.ADDED_FORMULAE}`)
+            test_bot_formulae_args.push(`--deleted-formulae=${process.env.DELETED_FORMULAE}`)
```
Comment (carlocab) [inline .github/workflows/tests.yml:160]: Do we want to add quotes here to make sure the contents of these environment variables aren't interpreted by the shell?

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -151,12 +155,12 @@ jobs:
...
 
             const test_bot_dependents_args = ["--only-formulae-dependents", "--junit"]
-            test_bot_dependents_args.push('--testing-formulae=${{needs.tap_syntax.outputs.testing_formulae}}')
+            test_bot_dependents_args.push(`--testing-formulae=${process.env.TESTING_FORMULAE}`)
```
Comment (carlocab) [inline .github/workflows/tests.yml:163]: Same here.

**Review by carlocab — DISMISSED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (Bo98) [reply .github/workflows/tests.yml:160]: Makes sense

**Review by SMillerDev — APPROVED** [review]

**Review by MikeMcQuaid — APPROVED** [review]
Works for me, thanks!

**Review by carlocab — APPROVED** [review]

**Pull request merged**